spec/octokit/client/repositories_spec.rb

```diff
@@ -484,6 +484,24 @@
     end
   end # .repository?
 
+  describe ".transfer_repository", :vcr do
+    it "repository transfer from myself to my organization" do
+      repository = @client.create_repository("an-repo", auto_init: true)
+      accept_header = Octokit::Preview::PREVIEW_TYPES[:transfer_repository]
+      @client.transfer_repository(repository.full_name, { new_owner: test_github_org, accept: accept_header })
+      assert_requested :post, github_url("/repos/#{repository.full_name}/transfer")
+
+      result = @client.repository?("#{test_github_org}/#{repository.name}")
+      expect(result).to be true
+
+      # cleanup
```

Comment:
Since this is in a separate `describe` we can use a separate `before` block for the setup and an `after` block for the cleanup.